packages/@sanity/base/src/presence/PresenceTooltip.tsx

```diff
@@ -1,40 +1,37 @@
-// @todo: remove the following line when part imports has been removed from this file
-///<reference types="@sanity/types/parts" />
-
-import {Tooltip, TooltipPlacement} from 'part:@sanity/components/tooltip'
 import React from 'react'
+import {Tooltip, Placement, Card, Flex} from '@sanity/ui'
 import {UserAvatar} from '../components/UserAvatar'
 import {FormFieldPresence} from './types'
 
-import styles from './PresenceTooltip.css'
+import {TextWrapper} from './PresenceTooltip.styled'
 
 interface PresenceTooltipProps {
   children?: React.ReactElement
   items: FormFieldPresence[]
-  placement?: TooltipPlacement
+  placement?: Placement
 }
 
 export function PresenceTooltip(props: PresenceTooltipProps) {
   const {children, items, placement} = props
 
   const content = (
-    <div className={styles.root}>
+    <Card radius={2} padding={2}>
       {items.map((item) => (
-        <div className={styles.item} key={item.user.id}>
-          <div className={styles.avatarContainer}>
-            <UserAvatar user={item.user} status="online" />
-          </div>
+        <Card key={item.user.id} padding={1}>
```

Comment:
Same here – this should be `<Box key={...`

Reply:
Done!